pkg/runtime.native.System.IO.Ports/runtime.native.System.IO.Ports.pkgproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <SystemIOPortsNativePath>$(ArtifactsBinDir)\native\$(TargetGroup)-$(DefaultOSGroup)-$(Configuration)-$(ArchGroup)</SystemIOPortsNativePath>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(OS)' != 'Windows_NT' and '$(PackageTargetRuntime)' != ''">
+    <NativeFile Include="$(SystemIOPortsNativePath)\System.IO.Ports.Native$(LibraryFileExtension)" >
+      <TargetPath>runtimes/$(PackageRID)/native</TargetPath>
+    </NativeFile>
+  </ItemGroup>
```

Comment:
You are missing a dependency item similar to the following:
https://github.com/dotnet/corefx/blob/3685052201c95462f210582be9fe380baf1b902e/pkg/frameworkPackage.targets#L26-L30

You can omit the TargetRuntime.  You’ll need a condition for ‘$(PackageTargetRuntime)’ == ‘’